### ISSUE TITLE
Hashcat consistently fails with CL_OUT_OF_HOST_MEMORY in mode 10500 with 32GB RAM

### ISSUE THREAD

**Issue by Missingmew** [hashcat/hashcat#2393]
As the title suggests, Im trying to run hashcat against a hash for PDF1.4-1.6 (as identified via example_hashes from the wiki). After some initial troubles with the mesa/clover opencl runtimes I installed the rocm opencl set and am now using bleeding edge hashcat from this repository.
Running either of the 3 provided examples or tools/test.sh works just fine without errors now, however, mode 10500 fails with the error message in the title. Since my system has 32GB of main RAM, I am ruling out an actual lack of system memory as the cause.
I tried running hc against different files to rule out broken sourcefiles but still get the same error.
Hashes were obtained using pdf2john, then stripping the filename prefix from the result.

System Info:
Debian unstable x64, kernel 5.6.0
GPU driver is upstream amdgpu via mesa 20.0.6
rocm-opencl reports in apt as version 2.0.0-rocm-rel-3.3-19-363509c8d
CPU is an i5-4690 with 32GB system RAM
GPU is a Vega 56

A sample file which fails for me can be found [here](https://www.novapdf.com/pdf-example-files-created-with-novapdf-kb.html) as "PDF Example - Password protection" (one of the first results from google for a usable sample).

**Comment by philsmd** [comment]
what is your command ? which version of hashcat do you use ?

can you test same command with your intel CPU (installing the Intel OpenCL Runtime as mentioned here: https://hashcat.net/hashcat ) ?

Thx

**Comment by philsmd** [comment]
any update here ?

We didn't get any responds here at all.

Did you already fix the problem ?

If we don't get any more feedback and detailed explanation of the problem and answers for the questions from above, we might need to close this issue as INVALID.
Please reply . Thx

**Comment by Missingmew** [comment]
Im sorry for missing your comments here (I didnt get any notifications by email for some reason).
As for the used hashcat version, I used the at the time current version on master. The exact command is unfortunately lost to time, but I specified the absolute minimum required.
I also currently do not have access to the machine I was using at the time so I cannot retry with the intel runtime. The usecase I had at the time also solved itself thanks to someone remembering the password of the file in question so there is no demand for an immediate fix from me right now.
